libr/bin/format/mach0/mach0.c

```diff
@@ -2892,8 +2892,56 @@ static int reloc_comparator(struct reloc_t *a, struct reloc_t *b) {
 	return a->addr - b->addr;
 }
 
+static void parse_relocation_info (struct MACH0_(obj_t) *bin, RSkipList * relocs, ut32 offset, ut32 num) {
+	if (!num || !offset) {
+		return;
+	}
+
+	ut64 total_size = num * sizeof (struct relocation_info);
+	struct relocation_info *info = calloc (1, total_size);
```

Comment:
better use calloc (num, sizeof (...)) instead of the resuslt of a*b, it's safer to integer overflows

Reply:
R_news0 even better